### PR TITLE
demonstrate simple daemonset to fix master kubelet bootstrap creds

### PR DESCRIPTION
**Pull request by deads2k**
related to https://bugzilla.redhat.com/show_bug.cgi?id=1735180

simplest solution I could come up with.  this fixes the "my cluster doesn't sleep issue" and fixes the disaster recovery updates.

/assign @sjenning @rphillips

### CODE REVIEW

**Comment by rphillips** [comment]
As discusses in slack, the kubeconfig is ignition managed and overwriting the kubeconfig in this manner will cause the MCD to validate the contents of the file and tainting the node since they differ.

**Comment by deads2k** [comment]
> As discusses in slack, the kubeconfig is ignition managed and overwriting the kubeconfig in this manner will cause the MCD to validate the contents of the file and tainting the node since they differ.

I think we just spoke again in slack and this wasn't the case.  The file doesn't appear to be managed and already has "bad" content.

File: install/0000_80_machine-config-operator_04_bootstrapdaemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+              kubectl config set-credentials "kubelet" --token="$token" >/dev/null
+              ca_crt="$(mktemp)"; echo "$ca_crt_data" > $ca_crt
+              kubectl config set-cluster $cluster --server="$intapi" --certificate-authority="$ca_crt" --embed-certs >/dev/null
+              kubectl config set-context kubelet --cluster="$cluster" --user="kubelet" >/dev/null
+              kubectl config use-context kubelet >/dev/null
+
+              echo "----------------------------------------------------------------------"
+              echo "Print kubeconfig"
+              echo "----------------------------------------------------------------------"
+              cat "$KUBECONFIG"
```
Comment (sjenning) [inline install/0000_80_machine-config-operator_04_bootstrapdaemonset.yaml]: probably want to remove this before merging right? @deads2k was this a kinda DO NOT MERGE experimental PR or is it suitable for merging consideration?

**Pull request closed without merging**